go/stellar/stellarsvc/frontend_test.go

```diff
@@ -241,14 +241,25 @@ func TestLinkNewWalletAccountLocal(t *testing.T) {
 	require.NoError(t, err)
```

Comment:
btw you can get of this CreateWallet thing in tests now because
1. creating the initial wallet is synchronous now
2. most wallet RPCs will create one if they need one
and it's nice to test that stuff

Reply:
good idea!